pkg/model/model.go

```diff
@@ -24,11 +24,13 @@ import (
 // MaskEngine is a masking algorithm
 type MaskEngine interface {
 	Mask(Entry, ...Dictionary) (Entry, error)
+	Name() string
```

Comment:
Maybe Name is not the good name... String()

Reply:
Yes, signature `String() string` if you want to implement the [fmt.Stringer](https://tour.golang.org/methods/17) interface (and then get compatibility with all method from fmt package)